diff -p -up gnome-system-tools-2.30.1/src/time/Makefile.am~ gnome-system-tools-2.30.1/src/time/Makefile.am
--- gnome-system-tools-2.30.1/src/time/Makefile.am~	2009-09-21 14:59:55.000000000 +0200
+++ gnome-system-tools-2.30.1/src/time/Makefile.am	2010-04-27 09:07:47.000000000 +0200
@@ -15,7 +15,7 @@ INCLUDES += $(GST_TOOL_CFLAGS) $(DBUS_CF
 CLEANFILES = $(GST_TOOL_CLEANFILES)
 EXTRA_DIST = $(GST_TOOL_EXTRA_DIST)
 
-time_admin_LDADD = $(GST_TOOL_LIBS) $(DBUS_LIBS) ./e-map/libemap.a
+time_admin_LDADD = $(GST_TOOL_LIBS) $(DBUS_LIBS) ./e-map/libemap.a -lm
 time_admin_DEPENDENCIES = $(GST_TOOL_DEPENDENCIES) ./e-map/libemap.a
 time_admin_SOURCES = \
 	time-tool.c	time-tool.h \
