diff -p -up gnome-system-tools-2.31.1/config.h.in~ gnome-system-tools-2.31.1/config.h.in
diff -p -up gnome-system-tools-2.31.1/src/common/Makefile.am~ gnome-system-tools-2.31.1/src/common/Makefile.am
--- gnome-system-tools-2.31.1/src/common/Makefile.am~	2010-05-24 19:09:57.000000000 +0200
+++ gnome-system-tools-2.31.1/src/common/Makefile.am	2010-05-25 12:33:02.000000000 +0200
@@ -4,6 +4,7 @@ noinst_LIBRARIES = libsetuptool.a
 
 INCLUDES += \
 	$(GST_CFLAGS) \
+	$(DBUS_CFLAGS) \
 	-DDATADIR=\""$(datadir)"\" \
 	-DLIBDIR=\""$(libdir)"\" \
 	-DSYSCONFDIR=\""$(sysconfdir)"\" \
diff -p -up gnome-system-tools-2.31.1/src/services/Makefile.am~ gnome-system-tools-2.31.1/src/services/Makefile.am
--- gnome-system-tools-2.31.1/src/services/Makefile.am~	2009-09-21 14:59:55.000000000 +0200
+++ gnome-system-tools-2.31.1/src/services/Makefile.am	2010-05-25 12:37:18.000000000 +0200
@@ -11,7 +11,7 @@ toolname = services
 bin_PROGRAMS = services-admin
 
 ## SUBDIRS =
-INCLUDES += $(GST_TOOL_CFLAGS)
+INCLUDES += $(GST_TOOL_CFLAGS) $(DBUS_CFLAGS)
 CLEANFILES = $(GST_TOOL_CLEANFILES)
 EXTRA_DIST = $(GST_TOOL_EXTRA_DIST)
 
diff -p -up gnome-system-tools-2.31.1/src/shares/Makefile.am~ gnome-system-tools-2.31.1/src/shares/Makefile.am
--- gnome-system-tools-2.31.1/src/shares/Makefile.am~	2009-09-21 14:59:55.000000000 +0200
+++ gnome-system-tools-2.31.1/src/shares/Makefile.am	2010-05-25 12:39:52.000000000 +0200
@@ -14,7 +14,7 @@ if BUILD_NAUTILUS
 SUBDIRS = nautilus
 endif
 
-INCLUDES += $(GST_TOOL_CFLAGS)
+INCLUDES += $(GST_TOOL_CFLAGS) $(DBUS_CFLAGS)
 CLEANFILES = $(GST_TOOL_CLEANFILES)
 EXTRA_DIST = $(GST_TOOL_EXTRA_DIST)
 
diff -p -up gnome-system-tools-2.31.1/src/shares/nautilus/Makefile.am~ gnome-system-tools-2.31.1/src/shares/nautilus/Makefile.am
--- gnome-system-tools-2.31.1/src/shares/nautilus/Makefile.am~	2010-01-11 20:27:00.000000000 +0100
+++ gnome-system-tools-2.31.1/src/shares/nautilus/Makefile.am	2010-05-25 12:41:51.000000000 +0200
@@ -1,4 +1,5 @@
 INCLUDES = $(GST_TOOL_CFLAGS)				\
+	$(DBUS_CFLAGS)					\
 	$(NAUTILUS_CFLAGS)				\
 	-DDATADIR=\"$(datadir)\"			\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\"	\
diff -p -up gnome-system-tools-2.31.1/src/users/Makefile.am~ gnome-system-tools-2.31.1/src/users/Makefile.am
--- gnome-system-tools-2.31.1/src/users/Makefile.am~	2010-02-10 17:29:37.000000000 +0100
+++ gnome-system-tools-2.31.1/src/users/Makefile.am	2010-05-25 12:45:23.000000000 +0200
@@ -11,7 +11,7 @@ toolname = users
 bin_PROGRAMS = users-admin
 
 SUBDIRS = 
-INCLUDES += $(GST_TOOL_CFLAGS)
+INCLUDES += $(GST_TOOL_CFLAGS) $(DBUS_CFLAGS)
 
 users_admin_LDADD = $(GST_TOOL_LIBS) 
 users_admin_DEPENDENCIES = $(GST_TOOL_DEPENDENCIES) 
